File: Jathin/easy/sample.js
console.log("selection sort")

let arr=[6,5,3,35,46,75,23,3,4687,564,2342,457,568,856];
let lens=(arr.length)
for(let i=0;i<lens;i++){
    let min = i;
    for(let j=i+1;j<lens;j++){
            if(arr[min] > arr[j]){
                    min =j;
            }
          let k = arr[min]
            arr[min]=arr[i]
            arr[i]=k;
        }

    }

    for(let i=0;i<lens;i++) console.log(arr[i])

File: Jathin/easy/sample.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int arr[]={4453,43,23,435,436,3457,42,12,32,12,323,43,6,5687,58};
    int sizes = sizeof(arr)/ sizeof(arr[1]);
    for(int i=0;i<sizes;i++){
        int min = i;
        for(int j=i;j<sizes;j++){
            if(arr[min]>arr[j]) min = j;
        }
    swap(arr[min],arr[i]);
    } 

for(int i=0;i<sizes;i++) cout<<arr[i]<<" ";
}